docs/en/sql-reference/statements/alter/statistics.md

```diff
@@ -25,7 +25,7 @@ Also, they are replicated, syncing statistics metadata via ZooKeeper.
 There is an example adding two statistics types to two columns:
 
 ```
-ALTER TABLE t1 MODIFY STATISTICS c, d TYPE TDigest, Uniq;
+ALTER TABLE t1 MODIFY STATISTICS c, d TYPE TDigest, Uniq, CMSketch;
```

Comment:
Can we please call the new statistics type "CountMin"? That will be more intuitive than "CMSketch".

Reply:
Please make the same adjustment in the code, i.e. class names "CountMinSketch" instead of "CMSketch".